### PR TITLE
OKTA-793295 - Add note to create a child org section of Aerial guide

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added a note about consent to Aerial guide
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-793295](https://oktainc.atlassian.net/browse/OKTA-793295)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md](https://github.com/okta/okta-developer-docs/blob/7f9346b1a8243c3515680acc0a4d48c41d8cd654/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmanage-orgs-okta-aerial%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/46967c7e-fa47-43e8-a924-a82eec6f68dc) | 94  | 80  | 100  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.caf8f9f7-93b1-493b-9e19-26da6036516b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md [hunk 1]
```diff
@@ -145,7 +145,9 @@ Authorization: Bearer ${access_token}
 
 Create a child org of the parent org using the Org creator API. This creates a child org with features synced from the parent org. In the API response, you receive an API token tied to the super admin. Use the token to provision more resources on the child org like policies, apps, or groups.
 
-This isn't the token that's used for Okta Aerial. The API token that the Org creator API creates has the same automatic expiration and deactivation as API tokens created using the [Admin Console](/docs/guides/create-an-api-token/main/#token-expiration-and-deactivation).
+>**Note:** Before creating a child org, ensure you give consent to the parent org with the [Grant Okta Aerial access to your Org API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/OrgSettingSupport/#tag/OrgSettingSupport/operation/grantAerialConsent).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md:148]:
```suggestion
>**Note:** Before creating a child org, ensure that you give consent to the parent org with the [Grant Okta Aerial access to your Org API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/OrgSettingSupport/#tag/OrgSettingSupport/operation/grantAerialConsent).
```

File: packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md [hunk 2]
```diff
@@ -145,7 +145,9 @@ Authorization: Bearer ${access_token}
...
+
+The API token isn't the token that's used for Okta Aerial. The token that the Org creator API creates has the same automatic expiration and deactivation as API tokens created using the [Admin Console](/docs/guides/create-an-api-token/main/#token-expiration-and-deactivation).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md:150]: It wasn't clear to me which API token was being talked about at the beginning of the sentence. So, I switched things around. LMK if you have questions.
```suggestion
The API token created using the [Admin Console](/docs/guides/create-an-api-token/main/#token-expiration-and-deactivation) isn't the token that's used for Okta Aerial. However, the token that the Org creator API creates has the same automatic expiration and deactivation.
```

**Review by susanharper-okta — COMMENTED** [review]
couple edits/comments

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md](https://github.com/okta/okta-developer-docs/blob/85af4a764d3fc7aed2bcdc67c00edbf71e0bb032/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmanage-orgs-okta-aerial%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5142aca5-c411-4606-b6b3-69a7fae9fc8a) | 94  | 80  | 100  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b3126677-e002-425a-b821-b55111e792c6?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md](https://github.com/okta/okta-developer-docs/blob/a11d1411d47c577350da133ca038a19c13572dc6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmanage-orgs-okta-aerial%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3cb9b264-b5ef-4b52-8c77-d408a97475d4) | 94  | 80  | 94  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.c6883811-b9f4-40a2-b4e7-8dc4b5d6186e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**